rpc/errors_test.go

```diff
@@ -0,0 +1,24 @@
+package rpc
+
+import (
+	"context"
+	"testing"
+
+	"github.com/test-go/testify/require"
+)
+
+func TestRPCError(t *testing.T) {
+	if testEnv == "mock" {
```

Comment:
what is `testEnv` variable? wdyt about chaing it to more go way: using go tags for files

Reply:
> Happy to open another issue fr a refactor (could bea good first issue?)

Yep, created the issue - https://github.com/NethermindEth/starknet.go/issues/543